elements/lisk-chain/src/chain.ts

```diff
@@ -346,9 +348,14 @@ export class Chain {
 			toHeight,
 		);
 
+		const lastReward = this.blockReward.calculateReward(
+			lastBlockHeaders[0]?.height ?? 1,
+		);
+
 		return new StateStore(this.storage, {
 			networkIdentifier: this._networkIdentifier,
 			lastBlockHeaders,
+			lastReward,
```

Comment:
```suggestion
		const lastBlockReward = this.blockReward.calculateReward(
			lastBlockHeaders[0]?.height ?? 1,
		);

		return new StateStore(this.storage, {
			networkIdentifier: this._networkIdentifier,
			lastBlockHeaders,
			lastBlockReward,
```
it will be more consistent like `lastBlockHeaders`